### PR TITLE
Remove TL_CRON usage from comments-bundle

### PR DESCRIPTION
**Pull request by fritzmg**
See https://github.com/contao/contao/issues/4513

### CODE REVIEW

File: comments-bundle/src/Cron/PurgeSubscriptionsCron.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        }
+
+        /** @var CommentsNotifyModel $subscription */
+        foreach ($subscriptions as $subscription) {
+            $subscription->delete();
+        }
+
+        if (null !== $this->logger) {
+            $this->logger->info('Purged the unactivated comment subscriptions');
+        }
```
Comment (ausi) [inline comments-bundle/src/Cron/PurgeSubscriptionsCron.php]:
```suggestion
        $this->logger?->info('Purged the unactivated comment subscriptions');
```

File: comments-bundle/src/Resources/contao/classes/Comments.php [hunk 2]
```diff
@@ -506,9 +506,13 @@ public function convertLineFeeds($strComment)
 
 	/**
 	 * Purge subscriptions that have not been activated within 24 hours
+	 *
+	 * @deprecated Deprecated since Contao 5.0, to be removed in Contao 6.0.
 	 */
 	public function purgeSubscriptions()
 	{
+		trigger_deprecation('contao/comments-bundle', '5.0', 'Calling "%s::%s()" has been deprecated and will no longer work in Contao 6.0.', __CLASS__, __METHOD__);
```
Comment (ausi) [inline comments-bundle/src/Resources/contao/classes/Comments.php]:
```suggestion
		trigger_deprecation('contao/comments-bundle', '5.0', 'Calling "%s()" has been deprecated and will no longer work in Contao 6.0.', __METHOD__);
```

**Review by ausi — DISMISSED** [review]

**Review by m-vo — DISMISSED** [review]

**Review by Toflar — DISMISSED** [review]

**Review by ausi — DISMISSED** [review]

**Review by aschempp — DISMISSED** [review]

**Review by leofeyer — CHANGES_REQUESTED** [review]
The changes are OK but the comments bundle does not have unit tests set up, so either you set them up or you remove the test class again.

**Comment by leofeyer** [comment]
You will also have to add a `phpunit.xml.dist` file in the bundle folder, adjust the workflow file and add the bundle to the list of split bundles that are tested in the CI. Just adding it in the root `phpunit.xml.dist` file is not enough, I‘m afraid.

**Comment by fritzmg** [comment]
@leofeyer should be RTM.

**Review by m-vo — DISMISSED** [review]

**Review by ausi — DISMISSED** [review]

File: comments-bundle/tests/Cron/PurgeSubscriptionsCronTest.php [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CommentsBundle\Tests\EventListener;
```
Comment (leofeyer) [inline comments-bundle/tests/Cron/PurgeSubscriptionsCronTest.php]:
```suggestion
namespace Contao\CommentsBundle\Tests\Cron;
```

The namespace seems to be wrong.

**Review by leofeyer — CHANGES_REQUESTED** [review]

**Review by ausi — APPROVED** [review]

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @fritzmg.

**Pull request merged**